apps/app/components/ui/tooltip.tsx

```diff
@@ -5,7 +5,7 @@ import { Tooltip2 } from "@blueprintjs/popover2";
 type Props = {
   tooltipHeading?: string;
   tooltipContent: string | JSX.Element;
-  triangle?: boolean;
+  hoverOpenDelay?: number;
```

Comment:
The prop name doesn't need to be this complex, you can rename it to just `delay`

Reply:
I used this name since it is the Tooltip2 prop name.
While I agree with you, this might pose an issue in the future if we decide to add the other time based controls that Tooltip2 uses. It has 3:
* `hoverOpenDelay`
* `hoverCloseDelay`
* `transitionDuration`

I would say I could change it to `openDelay` at least wdyt?

Reply:
Yes, that'd work.